### ISSUE TITLE
XML

### ISSUE THREAD

**Issue by gman2691** [panzarino/mlbgame#24]
The examples of your program on GitHub have an XMLSyntax Error

**Comment by zachpanz88** [comment]
@gman2691 can you please provide a direct example of where this occurs

**Comment by gman2691** [comment]
The teaser code that you posted to find all of the scores in the Mets games.

**Comment by zachpanz88** [comment]
@gman2691 I'm not getting any error, can you please post the error response you are getting?

**Comment by jimluke827** [comment]
I get this error lxml.etree.XMLSyntaxError: Start tag expected, '<' not found, line 1, column 1

**Comment by jimluke827** [comment]
full copy:

Traceback (most recent call last):
  File "C:/......../main.py", line 5, in <module>
    month = mlbgame.games(2015, 6, home="Mets")
  File "C:\.........\__init__.py", line 205, in games
    game = day(i, y, x, home=home, away=away)
  File "C:\..........\__init__.py", line 175, in day
    data = mlbgame.game.scoreboard(year, month, day, home=home, away=away)
  File "C:\.................\game.py", line 17, in scoreboard
    parsed = etree.parse(data)
  File "src\lxml\lxml.etree.pyx", line 3427, in lxml.etree.parse (src\lxml\lxml.etree.c:81101)
  File "src\lxml\parser.pxi", line 1811, in lxml.etree._parseDocument (src\lxml\lxml.etree.c:117832)
  File "src\lxml\parser.pxi", line 1837, in lxml.etree._parseDocumentFromURL (src\lxml\lxml.etree.c:118179)
  File "src\lxml\parser.pxi", line 1741, in lxml.etree._parseDocFromFile (src\lxml\lxml.etree.c:117091)
  File "src\lxml\parser.pxi", line 1138, in lxml.etree._BaseParser._parseDocFromFile (src\lxml\lxml.etree.c:111637)
  File "src\lxml\parser.pxi", line 595, in lxml.etree._ParserContext._handleParseResultDoc (src\lxml\lxml.etree.c:105093)
  File "src\lxml\parser.pxi", line 706, in lxml.etree._handleParseResult (src\lxml\lxml.etree.c:106801)
  File "src\lxml\parser.pxi", line 635, in lxml.etree._raiseParseError (src\lxml\lxml.etree.c:105655)
  File "file:/C:/.................../scoreboard.xml.gz", line 1
lxml.etree.XMLSyntaxError: Start tag expected, '<' not found, line 1, column 1

Process finished with exit code 1

**Comment by zachpanz88** [comment]
@jimluke827 I'm not getting that error at all, it could be an issue with lxml. Can you try updating both mlbgame and lxml to the latest version?

**Comment by jimluke827** [comment]
@zachpanz88 I am at the latest version of each.

**Comment by jimluke827** [comment]
@zachpanz88 I was using Python 3, I used python 2 and had a ton of trouble getting libxml2 to install, but once I got that all set everything is working

@gman2691 see above if you are using wrong version of python

**Comment by zachpanz88** [comment]
@jimluke827 that might have been the issue. I've only been testing on python 2, so I'll check out the issue on python 3.

**Comment by zachpanz88** [comment]
Again, I do not seem to get the error when I am testing with python 3.

> File "file:/C:/.................../scoreboard.xml.gz", line 1

@jimluke827 Is there any way you could fill in part of that "......" so I can diagnose what day the information is coming from? I just need the "month_06/day_xx/" part.

Just to be clear, we are talking about this example, correct?
```python
from __future__ import print_function
import mlbgame

month = mlbgame.games(2015, 6, home="Mets")
games = mlbgame.combine_games(month)
for game in games:
    print(game)
```

**Comment by jimluke827** [comment]
@zachpanz88 I don't have the error anymore, but I just used the exact examples that were shown in the readme, neither worked and both gave the same error.

**Comment by zachpanz88** [comment]
@jimluke827 I noticed from your error messages that you are using Windows. I have been testing everything on Linux, which handles .tar.gz much better than Windows. The problem could have arisen from Windows not being able to handle the compressed files, so I'll run some tests on Windows.

**Comment by rhoffer21** [comment]
Expanding on this, I also had the same issue.  I believe it is related to the .gz compressed xml files in a windows environment. I was eventually able to remedy the issue by moving the archived gameday-data to a backup folder which forced the wrapper to make calls to the server instead of using the archived xml data. 

I unfortunately don't have a better work around to offer at this time, other than moving the saved data.

**Comment by zachpanz88** [comment]
@rhoffer21 I will try to find a better compression format for windows (probably zip). I will also build a delete function into the update module that will allow you to remove all downloaded data.

**Comment by panzarino** [comment]
Released in [2.3.2](https://github.com/panzarino/mlbgame/releases/tag/v2.3.2).
